packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -66,12 +94,8 @@ class MemoryController extends DisposableController
     });
   }
 
-  /// The controller is late to enable test injection.
-  late final DiffPaneController diffPaneController;
-
-  final profilePaneController = ProfilePaneController();
-
-  final tracingPaneController = TracingPaneController();
+  /// Sub-controllers of memory controller.
+  late final MemoryChildControllers children;
```

Comment:
nit: rename to `featureControllers`

Reply:
featureControllers is too long. how about just `controllers' ?